src/no/runsafe/framework/minecraft/entity/animals/RunsafeSheep.java

```diff
@@ -48,5 +56,6 @@ public Sound getIdleSound()
 		return Sound.Creature.Sheep.Idle;
 	}
 
+	private Random random = new Random();
```

Comment:
Something feels wrong about keeping a rng on every sheep instance just to facilitate `setRandomColour()` - make it a static?